### PR TITLE
Turkish Update 4.2.2

### PR DESCRIPTION
**Pull request by HuseyinFiliz**
Based on my earlier message, I would like to inform you that I have made the necessary corrections to the translation, including the minor errors and the oversight in not including the contributions of the other Turkish translator.

### CODE REVIEW

**Comment by TrianguloY** [comment]
Perfect, thank you for the translation update! 

I'll review and merge it on the next days, feel free to update it if necessary (or create new prs if you discover a problem after merging)

**Comment by HuseyinFiliz** [comment]
I dont plan to make any further changes until the update is released, unless a new addition is made in the original language that requires a translation update or until the update is featured in the Play Store's 'What's New' seçtin (changelog). Would joining the beta program enable me to receive the update faster? Also, is there a way to download the update as an APK file instead of waiting for it to become available on the Play Store? Furthermore, could you provide me with a code to test the donation version?

**Comment by TrianguloY** [comment]
> Would joining the beta program enable me to receive the update faster? 

Yes, I always test new version as beta first (they rarely fail but just in case)

> Also, is there a way to download the update as an APK file instead of waiting for it to become available on the Play Store? 

Nothing automated, but I can build it and post it here if you want.

> Furthermore, could you provide me with a code to test the donation version?

Sure! Send me an email to send it to you there.

**Comment by HuseyinFiliz** [comment]
If you can share the update here before it's released on the Play Store, I can test it for you at the same time. That would be great for me too. Thank you for your interest, I have sent an email.

**Comment by TrianguloY** [comment]
[app-original-evaluation.zip](https://github.com/TrianguloY/OpenInWhatsapp_Translation/files/11347664/app-original-evaluation.zip)

Note: this is an unsigned debug version that can be installed without removing the original from play store (it's considered a different app). It may fail, and anyone could theoretically create updates for it (which may be dangerous if you install an update from somewhere else), so use it to test the update and then I suggest to uninstall it.
There are no changes other than the ones from this pr.

**Comment by HuseyinFiliz** [comment]
Wow, I didn't expect it to turn out this well. I was a bit apprehensive about making mistakes since this was my first translation, but when I checked it in the application, everything looked perfect. Thank you for the prompt feedback.

Note: I didn't have to remove the previous application, so it is still installed. Now there are two different applications :)

**Comment by TrianguloY** [comment]
Perfect, I'll merge it tonight (or tomorrow I hope).

As for the two apps, yes, perhaps my comment wasn't well written. It is considered a different application, that's why you didn't need to uninstall the play store one. 
This version however isn't signed so anyone could create malicious updates for it. Just don't install random apks from other sources and you are good, but still I recommend to uninstall it when you finish reviewing it (it will become obsolete once the normal app is updated in any case).

File: PlayStore/tr-TR.xml [hunk 1]
```diff
@@ -6,62 +6,55 @@ Instructions for manual translation:
...
-Rehberinizde kişi oluşturulmaz. Sohbet başlatmak için numarayı rehbere kaydetmekle uğraşmazsınız. Yalnızca uygulamayı açın, numarayı girin, butona basın ve sohbet açılsın. (Numara mevcutsa, aksi takdirde WhatsApp bunu belirtecektir.)
+    <!-- Preferably use the name from https://faq.whatsapp.com/general/chats/how-to-use-click-to-chat/ in your language -->
+    <title max_chars="30">Tıkla Sohbet Et</title>
 
-Özellikle şu gibi durumlarda işe yarar:
-- Bilinmeyen bir numara sizi aradı ve siz de bu numaranın WhatsApp'ı olup olmadığını mı merak ediyorsunuz?
-- Birine mesaj göndermek istiyorsunuz ama numarasını rehbere kaydetmek istemiyor musunuz?
-- Kendinizle nr sohbet başlatmak mı istiyorsunuz?
-Sadece uygulamayı açın ve numarayı girin. (Ülke konuyla birlikte.)
+    <description_short max_chars="80">Rehbere kaydetmeden bir numarayla WhatsApp'tan konuşmanızı sağlayan küçük ve reklamsız bir uygulama.</description_short>
```
Comment (TrianguloY) [inline PlayStore/tr-TR.xml:18]: This description_short is longer than 80 characters. Can you reduce it? It can't be updated if longer.

**Review by TrianguloY — CHANGES_REQUESTED** [review]

**Comment by HuseyinFiliz** [comment]
Can you make the change?

<description_short max_chars="80">Rehbere kaydetmeden WhatsApp'tan konuşmanızı sağlar. Küçük ve reklamsız.</description_short>

**Comment by TrianguloY** [comment]
@HuseyinFiliz I sent you the code by email yesterday, did you not receive it? 
I've sent it again now.

**Comment by HuseyinFiliz** [comment]
It went to my email spam folder. Thank you.

**Comment by TrianguloY** [comment]
> It went to my email spam folder. Thank you.

Oh, I don't like that my emails are considered spam (I rarely send them), maybe because of the code? If you can mark it as no-spam that would help, I guess.

**Comment by HuseyinFiliz** [comment]
I have marked it as not spam. Thank you for the code :)

**Pull request merged**